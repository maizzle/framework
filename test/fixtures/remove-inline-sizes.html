<!DOCTYPE html>
<html>
<head>
  <if condition="page.css">
    <style>{{{ page.css }}}</style>
  </if>
</head>
<body>
  <table class="w-full">
    <tr>
      <td class="h-full">Test</td>
    </tr>
  </table>
</body>
</html>
