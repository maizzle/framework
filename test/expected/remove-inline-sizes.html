<!DOCTYPE html>
<html>
<head>


  </head>
<body>
  <table class="w-full" width="100%" cellpadding="0" cellspacing="0" role="presentation">
    <tr>
      <td class="h-full" height="100%">Test</td>
    </tr>
  </table>
</body>
</html>
